lib/editor/util/map.js

```diff
@@ -288,7 +288,7 @@ export async function recalculateShape ({
   index: number,
   newPoint?: LatLng,
   pattern: Pattern,
-  patternCoordinates: Coordinates,
+  patternCoordinates: Array<Coordinates>,
```

Comment:
Another great candidate for adding tests.  Especially since there is already a file `lib/editor/util/__tests__/map.js_hold` with test cases for this exact function.